Ical.Net/DataTypes/RecurrencePattern.cs

```diff
@@ -37,13 +44,19 @@ public CalDateTime? Until
         }
     }
 
+    /// <summary>
+    /// Specifies the number of occurrences of the recurrence (optional).
+    /// This property <b>must be null</b> if the <see cref="Until"/> property is set.
+    /// </summary>
     public int? Count { get; set; }
 
+
     /// <summary>
-    /// Specifies how often the recurrence should repeat.
-    /// - 1 = every
-    /// - 2 = every second
-    /// - 3 = every third
+    /// Specifies the interval between occurrences of the recurrence. The default value is 1.
```

Comment:
That's not fully precise, as due to the other rule parts, more or less occurrences could happen per interval. This is what the RFC says, maybe just copy?

> The INTERVAL rule part contains a positive integer representing at
      which intervals the recurrence rule repeats.  The default value is
      "1", meaning every second for a SECONDLY rule, every minute for a
      MINUTELY rule, every hour for an HOURLY rule, every day for a
      DAILY rule, every week for a WEEKLY rule, every month for a
      MONTHLY rule, and every year for a YEARLY rule.  For example,
      within a DAILY rule, a value of "8" means every eight days.